cms/forms/widgets.py

```diff
@@ -291,11 +292,11 @@ def __init__(self, attrs=None, choices=(), app_configs={}):
         super(ApplicationConfigSelect, self).__init__(attrs, choices)
 
     def render(self, name, value, attrs=None, choices=()):
-        output = [super(ApplicationConfigSelect, self).render(name, value, attrs, choices)]
+        output = list(super(ApplicationConfigSelect, self).render(name, value, attrs, choices))
         output.append('<script>\n')
         output.append('var apphooks_configuration = {\n')
         for application, cms_app in self.app_configs.items():
-            output.append("'%s': [%s]," % (application, ",".join(["['%s', '%s']" % (config.pk, force_text(config)) for config in cms_app.get_configs()])))
+            output.append("'%s': [%s]," % (application, ",".join(["['%s', '%s']" % (config.pk, escape(config)) for config in cms_app.get_configs()])))  # noqa
```

Comment:
@yakky, @stefanfoulis good points. I'll check to see which is best (using just escapejs or both), and patch as required.